src/core/handle.ml

```diff
@@ -89,19 +90,20 @@ let handle_require_as : popt -> sig_state -> Path.t -> ident -> sig_state =
     This structure contains the list of the tactics to be executed, as well as
     the initial state of the proof.  The checking of the proof is then handled
     separately. Note that [Fatal] is raised in case of an error. *)
-let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
+let rec handle_cmd : sig_state -> p_command
+                     -> sig_state * proof_data option * inductive option =
```

Comment:
A less disruptive solution would be to factorize out the code for the case P_symbol by adding a function handle_symbol.

Reply:
Yes, I can do that but the main problem is the new type for the function "handle_cmd", not the fact that it becomes recursive, no?
But I must do this last modification.

Reply:
I think that you do not need to change the type and make it recursive. It is sufficient to add handle_symbol.

Reply:
OK, but how can I create my structure, I mean my record, to store enough data to build the inductive principle?
In fact, I followed the same process that with the constructor `P_theorem` where you need to store some information.